Testing/SystemTests/tests/framework/SANSILLAutoProcessTest.py

```diff
@@ -267,6 +367,22 @@ def setUp(self):
         config['logging.loggers.root.level'] = 'Warning'
         config.appendDataSearchSubDir('ILL/D11/')
 
+        # prepare mask for instrument edges first:
```

Comment:
I see you've added a new system test higher up. What's the reason for this CustomStitching system test changing as well?

Reply:
Well spotted as well, that seems to be introduced by merging master, where the test for solvent and custom stitching were mashed together. I removed the redundant mask definitions.